user-guide/data.rst

```diff
@@ -416,6 +416,110 @@ above.)
   Further information on using ``rsync`` can be found in the ``rsync`` manual
   (accessed via ``man rsync`` or at `man rsync <https://linux.die.net/man/1/rsync>`__).
 
+SSH data transfer example
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here we have a short example demonstrating transfer of data directly from ARCHER
+to ARCHER2. The first step will be to set up an SSH key for access to ARCHER2
+directly from ARCHER. Note that this process does require the ability to pass a
+file from ARCHER to ARCHER2 via a trusted location which has access to both
+machines. 
+
+First log in to ARCHER, and generate a new SSH key. To do this we use the
+following command:
+
+::
+
+  ssh-keygen -b 4096 -C "otbz01@archer -> otbz19@archer2" -f ~/.ssh/id_RSA_A2
+
+This generates a new 4096 bit RSA SSH key with the comment ``otbz01@archer ->
+otbz19archer2``, and stores the private key in the file ``~/.ssh/id_RSA_A2``,
+and the public key in a corresponding ``.pub`` file. During key generation
+process we are asked to enter a passphrase. SSH keys should *always* be
+passphrase protected, but this is especially important when they are kept on a
+publicly accessible machine such as ARCHER. Please *do not* leave the
+passphrase blank when setting up your SSH key. 
+
+Next we must transfer the public key to ARCHER2 and add it to the 
+`authorized_keys` file there. We must do this via a trusted intermediary which
+is able to access both machines - in this case a local computer. On the local
+(trusted) machine we first scp the *public* key from ARCHER to the local
+machine.
```

Comment:
@otbrown I think the key should be added via SAFE rather than by adding to authorized_keys. ARCHER2 (like Cirrus) supports multiple keys per account now.

Reply:
Nice! That simplifies things somewhat. I'll test and make the changes on Monday.

Reply:
...or tonight even.